packages/host/app/components/operator-mode/card-error-detail.gts

```diff
@@ -1,146 +1,57 @@
 }
 
 export default class CardErrorDetail extends Component<Signature> {
-  @tracked private showErrorDetail = false;
-  @service private declare commandService: CommandService;
-
-  private toggleDetail = () => (this.showErrorDetail = !this.showErrorDetail);
+  private get message() {
+    return this.args.error.message ?? undefined;
+  }
 
-  private viewInCodeMode = dropTask(async () => {
-    let switchSubmodeCommand = new SwitchSubmodeCommand(
-      this.commandService.commandContext,
-    );
-    await switchSubmodeCommand.execute({
-      submode: 'code',
-      codePath: `${this.args.error.id}.json`,
-    });
-  });
+  private get stack() {
+    return this.args.error.meta.stack ?? undefined;
+  }
 
   <template>
-    <Accordion
-      class='error-detail {{if this.showErrorDetail "open"}}'
-      ...attributes
-      as |A|
-    >
-      <A.Item
-        data-test-error-detail-toggle
-        @onClick={{fn this.toggleDetail 'schema'}}
-        @isOpen={{this.showErrorDetail}}
-      >
-        <:title>
-          <ExclamationCircle class='error-icon' />
-          {{if @headerText @headerText 'An error was encountered: '}}
-          <span data-test-error-title>
-            {{if @title @title @error.title}}
-          </span>
-        </:title>
-        <:content>
-          {{#if @viewInCodeMode}}
-            <div class='actions'>
-              <Button
-                data-test-view-in-code-mode-button
-                @kind='primary'
-                {{on 'click' (perform this.viewInCodeMode)}}
-              >View in Code Mode</Button>
-            </div>
-          {{/if}}
-          <div class='detail'>
-            <div class='detail-item'>
-              <div class='detail-title'>Details:</div>
-              <div
-                class='detail-contents'
-                data-test-error-detail
-              >{{@error.message}}</div>
-            </div>
-            {{#if @error.meta.stack}}
-              <div class='detail-item'>
-                <div class='detail-title'>Stack trace:</div>
-                <pre
-                  data-test-error-stack
-                  data-test-percy-hide
-                >
-{{@error.meta.stack}}
-                </pre>
-              </div>
-            {{/if}}
-          </div>
-        </:content>
-      </A.Item>
-    </Accordion>
+    <div class='error-detail' ...attributes>
+      <ErrorDisplay
+        @type='runtime'
+        @headerText={{@headerText}}
+        @message={{if this.message this.message @error.title}}
+        @stack={{this.stack}}
+        @fileToAttach={{@fileToFixWithAi}}
+        @viewInCodeMode={{@viewInCodeMode}}
+        @cardId={{@error.id}}
+      />
+    </div>
 
     <style scoped>
       .error-detail {
         flex: 1.5;
-        overflow: auto;
+        overflow: visible;
         margin-top: auto;
         max-height: fit-content;
-      }
-      .error-detail :deep(.accordion-item) {
-        height: auto;
+        margin: var(--boxel-sp);
```

Comment:
the `margin` in line 48 is overriding the `margin-top` in line 46. as a result, this component does not always appear at the bottom of the page.